### ISSUE TITLE
embed.cs.utah.edu/csmith is gone

### ISSUE THREAD

**Issue by amonakov** [csmith-project/csmith#149]
The http://embed.cs.utah.edu/csmith and especially http://embed.cs.utah.edu/csmith/using.html pages are gone. They were available a few days ago.

**Comment by eeide** [comment]
Thanks. This is due to some website reorganization at the University of Utah. I expect that the content will be relocated soon, to a different URL—but I do not yet know where.

**Comment by regehr** [comment]
anything of it that we want to keep should get relocated into this git repo, is my view, I'm not interested in maintaining a separate site for Csmith (or C-Reduce)

**Comment by amonakov** [comment]
Curious about the history there — the using.html page is pretty important, how come it was not part of this repo in the first place?

**Comment by regehr** [comment]
it's because Csmith lived in a private repo for several years before github even existed

**Comment by regehr** [comment]
(iirc)

**Comment by chenrui333** [comment]
Just found `http://embed.cs.utah.edu/csmith` is gone, should I expect it come back any time soon?

**Comment by eeide** [comment]
@chenrui333 No, `embed.cs.utah.edu` will not be coming back. I believe that @regehr is in the process of moving the content into this GitHub repository. See his comment above.

**Comment by brdjns** [comment]
Any news on this?
